xml/System.Numerics/Complex.xml

```diff
@@ -1930,40 +2000,61 @@
- [!code-vb[System.Numerics.Complex.Add#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.add/vb/add3.vb#3)]  
-  
- The addition of a complex number, a + bi, and a second complex number, c + di, takes the following form:  
-  
- (a + c) + (b + d)i  
-  
- If the method call results in an overflow in either the real or imaginary component, the value of that component is either <xref:System.Double.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>.  
-  
- Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Add%2A> method instead.  
-  
- The equivalent method for this operator is <xref:System.Numerics.Complex.Add%28System.Numerics.Complex%2CSystem.Numerics.Complex%29?displayProperty=nameWithType>  
-  
-## Examples  
- The following example illustrates addition with complex numbers.  
-  
- [!code-csharp[System.Numerics.Complex.Add#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.add/cs/add2.cs#2)]
- [!code-vb[System.Numerics.Complex.Add#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.add/vb/add2.vb#2)]  
-  
- ]]></format>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The addition of a complex number, a + bi, and a second complex number, c + di, takes the following form:
+
+(a + c) + (b + d)i
+
+If the addition results in an overflow in either the real or imaginary component, the value of that component is either <xref:System.Double.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>.
+
+Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Add%28System.Numerics.Complex%2CSystem.Double%29?displayProperty=nameWithType> equivalent method instead.
+
+          ]]></format>
         </remarks>
         <altmember cref="M:System.Numerics.Complex.Add(System.Numerics.Complex,System.Numerics.Complex)" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Division">
+      <Docs>
+        <summary>Divides a specified number by another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.Complex.op_Division%2A> operators allow performing division operation that involve complex numbers. They enable code such as the following:
+
+[!code-csharp[System.Numerics.Complex.Divide#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.divide/cs/divide2.cs#2)]
+[!code-vb[System.Numerics.Complex.Divide#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.divide/vb/divide2.vb#2)]
+
+The division of a complex number, a + bi, and a second complex number, c + di, takes the following form:
```

Comment:
The same comment applies as ealier. I'd remove the example divisions from here, since they apply to each overload. And I'd remove the overflow sentence and the custom operators sentence from the individual overloads.